scripts/api/data/licenses/licenseODbL-1.0.json

```diff
@@ -0,0 +1,11 @@
+{
+  "name": "ODbL-1.0",
+  "uri": "http://www.opendatacommons.org/licenses/odbl/1.0/",
+  "shortDescription": "Open Data Commons Open Database License v1.0",
```

Comment:
In DataCite Fabrica this is know as "ODC Open Database License v1.0". It seems to come from here: https://github.com/datacite/bracco/blob/df4ac2030774324adeaa72e2978c29dfb463ee3d/app/spdx.js#L3027

![Screenshot 2025-05-29 at 4 14 52 PM](https://github.com/user-attachments/assets/3e30ff83-aa5c-42ec-b75d-a3f81f987039)

In SPDX it's known as "Open Data Commons Open Database License v1.0" as shown above.

![Screenshot 2025-05-29 at 4 16 33 PM](https://github.com/user-attachments/assets/a03e57f0-d6fd-40ac-9f93-fec90de754a3)